aspnetcore/blazor/tooling.md

```diff
@@ -478,12 +726,6 @@ For more information, see the following resources in the Visual Studio documenta
 * [Introduction to projects and solutions](/visualstudio/get-started/tutorial-projects-solutions)
 * [What are solutions and projects in Visual Studio?](/visualstudio/ide/solutions-and-projects-in-visual-studio)
 
-## Use Visual Studio Code for cross-platform Blazor development
-
-[Visual Studio Code](https://code.visualstudio.com/) is an open source, cross-platform Integrated Development Environment (IDE) that can be used to develop Blazor apps. Use the [.NET CLI](/dotnet/core/tools/) to create a new Blazor app for development with Visual Studio Code. For more information, see the [Linux/macOS version of this article](?pivots=linux-macos).
-
-For more information on Visual Studio Code configuration and use, see the [Visual Studio Code documentation](https://code.visualstudio.com/docs).
-
 ## Blazor template options
```

Comment:
Above we go into a lot of detail on the different Blazor template options. Instead of showing all the variants, should we instead just show the default with no options and then reference the content in this section for details on all the various options?

Reply:
Some readers will be slightly annoyed by being thrown to a different section, but I'll change it.

Also, the template options section will need to receive the options content because it only lists the templates and recommends the *Project Structure* article or `-h|--help` for options details. I'll move at least some of the content down to this section, and I'll try to condense it a bit. Perhaps, a table would be a good choice for the content. I'll see what I can work out.

Reply:
***Done!*** 👍 ... Will be on the next commit.